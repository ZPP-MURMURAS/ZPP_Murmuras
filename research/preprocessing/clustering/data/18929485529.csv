ID,User ID,Time,I,Language,Application Name,Package Name,Class Name,Context,View ID,View Depth,View Class Name,Text,Description,Seen Timestamp,Is Visible,X 1,Y 1,X 2,Y 2
18929485529,165559,2024-09-04T10:55:25.287,1,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,0,de.penny.app.main.view.MainActivity,,,0,false,0,0,0,0
18929485529,165559,2024-09-04T10:55:25.287,2,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,android:id/content,2,android.widget.FrameLayout,,,1725440082464,true,0,0,1080,2400
18929485529,165559,2024-09-04T10:55:25.287,3,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 14.99,,1725440082464,true,339,833,498,874
18929485529,165559,2024-09-04T10:55:25.287,4,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,9.99,,1725440082464,true,356,884,482,960
18929485529,165559,2024-09-04T10:55:25.287,5,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440082464,true,63,986,125,1027
18929485529,165559,2024-09-04T10:55:25.287,6,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,JOHNNIE WALKER Red Label Blended Scotch,,1725440082464,true,63,1038,514,1134
18929485529,165559,2024-09-04T10:55:25.287,7,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,7 I",,1725440082464,true,63,1134,182,1188
18929485529,165559,2024-09-04T10:55:25.287,8,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 0.99,,1725440082464,true,862,832,1001,873
18929485529,165559,2024-09-04T10:55:25.287,9,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,0.75,,1725440082464,true,859,881,985,957
18929485529,165559,2024-09-04T10:55:25.287,10,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440082464,true,567,981,629,1022
18929485529,165559,2024-09-04T10:55:25.287,11,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,SAN MIGUEL Especial,,1725440082464,true,567,1030,926,1078
18929485529,165559,2024-09-04T10:55:25.287,12,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,5 I",,1725440082464,true,567,1072,686,1126
18929485529,165559,2024-09-04T10:55:25.287,13,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 2.99,,1725440082464,true,359,1569,498,1610
18929485529,165559,2024-09-04T10:55:25.287,14,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.79,,1725440082464,true,356,1620,482,1696
18929485529,165559,2024-09-04T10:55:25.287,15,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440082464,true,63,1722,125,1763
18929485529,165559,2024-09-04T10:55:25.287,16,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,FELIX Knabber Mix,,1725440082464,true,63,1774,385,1822
18929485529,165559,2024-09-04T10:55:25.287,17,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 200 g,,1725440082464,true,63,1822,204,1876
18929485529,165559,2024-09-04T10:55:25.287,18,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,3.89,,1725440082464,true,859,1619,985,1695
18929485529,165559,2024-09-04T10:55:25.287,19,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,Preisknaller,,1725440082464,true,567,1721,755,1762
18929485529,165559,2024-09-04T10:55:25.287,20,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,FELIX So gut wie es aussieht in Gelee,,1725440082464,true,567,1773,1017,1869
18929485529,165559,2024-09-04T10:55:25.287,21,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 12 x 85 g,,1725440082464,true,567,1869,771,1923
18929485529,165559,2024-09-04T10:55:25.287,22,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Sparen auf Top-Marken,,1725440082464,true,263,276,754,333
18929485529,165559,2024-09-04T10:55:25.287,23,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,ab 05.09. bis 07.09.,,1725440082464,true,263,333,566,379
18929485529,165559,2024-09-04T10:55:25.287,24,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Angebote,,1725440082464,true,44,2195,155,2227
18929485529,165559,2024-09-04T10:55:25.287,25,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Vorteile,,1725440082464,true,279,2197,362,2229
18929485529,165559,2024-09-04T10:55:25.287,26,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Einkaufsliste,,1725440082464,true,473,2195,608,2227
18929485529,165559,2024-09-04T10:55:25.287,27,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Vorteilscode,,1725440082464,true,694,2197,827,2229
18929485529,165559,2024-09-04T10:55:25.287,28,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Mein PENNY,,1725440082464,true,913,2195,1047,2227
18929485529,165559,2024-09-04T10:55:25.287,29,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,android:id/navigationBarBackground,1,android.view.View,,,1725440082464,false,0,2274,1080,2400
18929485529,165559,2024-09-04T10:55:25.287,30,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 14.99,,1725440082785,true,339,729,498,770
18929485529,165559,2024-09-04T10:55:25.287,31,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,9.99,,1725440082785,true,356,780,482,856
18929485529,165559,2024-09-04T10:55:25.287,32,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440082785,true,63,882,125,923
18929485529,165559,2024-09-04T10:55:25.287,33,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,JOHNNIE WALKER Red Label Blended Scotch,,1725440082785,true,63,934,514,1030
18929485529,165559,2024-09-04T10:55:25.287,34,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,7 I",,1725440082785,true,63,1030,182,1084
18929485529,165559,2024-09-04T10:55:25.287,35,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 0.99,,1725440082785,true,862,728,1001,769
18929485529,165559,2024-09-04T10:55:25.287,36,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,0.75,,1725440082785,true,859,779,985,855
18929485529,165559,2024-09-04T10:55:25.287,37,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440082785,true,567,881,629,922
18929485529,165559,2024-09-04T10:55:25.287,38,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,SAN MIGUEL Especial,,1725440082785,true,567,933,926,981
18929485529,165559,2024-09-04T10:55:25.287,39,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,5 I",,1725440082785,true,567,981,686,1035
18929485529,165559,2024-09-04T10:55:25.287,40,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 14.99,,1725440083429,true,339,478,498,519
18929485529,165559,2024-09-04T10:55:25.287,41,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,9.99,,1725440083429,true,356,529,482,605
18929485529,165559,2024-09-04T10:55:25.287,42,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440083429,true,63,631,125,672
18929485529,165559,2024-09-04T10:55:25.287,43,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,JOHNNIE WALKER Red Label Blended Scotch,,1725440083429,true,63,683,514,779
18929485529,165559,2024-09-04T10:55:25.287,44,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,7 I",,1725440083429,true,63,779,182,833
18929485529,165559,2024-09-04T10:55:25.287,45,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 0.99,,1725440083429,true,862,477,1001,518
18929485529,165559,2024-09-04T10:55:25.287,46,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,0.75,,1725440083429,true,859,528,985,604
18929485529,165559,2024-09-04T10:55:25.287,47,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440083429,true,567,630,629,671
18929485529,165559,2024-09-04T10:55:25.287,48,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,SAN MIGUEL Especial,,1725440083429,true,567,682,926,730
18929485529,165559,2024-09-04T10:55:25.287,49,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,5 I",,1725440083429,true,567,730,686,784
18929485529,165559,2024-09-04T10:55:25.287,50,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 2.99,,1725440083429,true,359,1318,498,1359
18929485529,165559,2024-09-04T10:55:25.287,51,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.79,,1725440083429,true,356,1369,482,1445
18929485529,165559,2024-09-04T10:55:25.287,52,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440083429,true,63,1471,125,1512
18929485529,165559,2024-09-04T10:55:25.287,53,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,FELIX Knabber Mix,,1725440083429,true,63,1523,385,1571
18929485529,165559,2024-09-04T10:55:25.287,54,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 200 g,,1725440083429,true,63,1571,204,1625
18929485529,165559,2024-09-04T10:55:25.287,55,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,3.89,,1725440083429,true,859,1272,985,1348
18929485529,165559,2024-09-04T10:55:25.287,56,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,Preisknaller,,1725440083429,true,567,1470,755,1511
18929485529,165559,2024-09-04T10:55:25.287,57,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,FELIX So gut wie es aussieht in Gelee,,1725440083429,true,567,1522,1017,1618
18929485529,165559,2024-09-04T10:55:25.287,58,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 12 x 85 g,,1725440083429,true,567,1618,771,1672
18929485529,165559,2024-09-04T10:55:25.287,59,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Framstag,,1725440083429,true,263,1744,462,1801
18929485529,165559,2024-09-04T10:55:25.287,60,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,ab 06.09. bis 07.09.,,1725440083429,true,263,1781,566,1827
18929485529,165559,2024-09-04T10:55:25.287,61,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 2.99,,1725440083734,true,359,435,498,476
18929485529,165559,2024-09-04T10:55:25.287,62,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.79,,1725440083734,true,356,486,482,562
18929485529,165559,2024-09-04T10:55:25.287,63,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440083734,true,63,588,125,629
18929485529,165559,2024-09-04T10:55:25.287,64,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,FELIX Knabber Mix,,1725440083734,true,63,640,385,688
18929485529,165559,2024-09-04T10:55:25.287,65,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 200 g,,1725440083734,true,63,688,204,742
18929485529,165559,2024-09-04T10:55:25.287,66,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,3.89,,1725440083734,true,859,485,985,561
18929485529,165559,2024-09-04T10:55:25.287,67,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,Preisknaller,,1725440083734,true,567,587,755,628
18929485529,165559,2024-09-04T10:55:25.287,68,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,FELIX So gut wie es aussieht in Gelee,,1725440083734,true,567,639,1017,735
18929485529,165559,2024-09-04T10:55:25.287,69,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 12 x 85 g,,1725440083734,true,567,735,771,789
18929485529,165559,2024-09-04T10:55:25.287,70,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Framstag,,1725440083734,true,263,1002,462,1059
18929485529,165559,2024-09-04T10:55:25.287,71,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,ab 06.09. bis 07.09.,,1725440083734,true,263,1059,566,1105
18929485529,165559,2024-09-04T10:55:25.287,72,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 8.99,,1725440083734,true,841,1728,980,1769
18929485529,165559,2024-09-04T10:55:25.287,73,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,5.85,,1725440083734,true,838,1779,964,1855
18929485529,165559,2024-09-04T10:55:25.287,74,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440083734,true,84,1881,146,1922
18929485529,165559,2024-09-04T10:55:25.287,75,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,CHANTRÉ Weinbrand,,1725440083734,true,84,1933,445,1981
18929485529,165559,2024-09-04T10:55:25.287,76,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,7 I",,1725440083734,true,84,1981,203,2035
18929485529,165559,2024-09-04T10:55:25.287,77,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Framstag,,1725440084039,true,263,425,462,482
18929485529,165559,2024-09-04T10:55:25.287,78,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,ab 06.09. bis 07.09.,,1725440084039,true,263,482,566,528
18929485529,165559,2024-09-04T10:55:25.287,79,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 8.99,,1725440084039,true,841,1151,980,1192
18929485529,165559,2024-09-04T10:55:25.287,80,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,5.85,,1725440084039,true,838,1202,964,1278
18929485529,165559,2024-09-04T10:55:25.287,81,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440084039,true,84,1304,146,1345
18929485529,165559,2024-09-04T10:55:25.287,82,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,CHANTRÉ Weinbrand,,1725440084039,true,84,1356,445,1404
18929485529,165559,2024-09-04T10:55:25.287,83,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,7 I",,1725440084039,true,84,1404,203,1458
18929485529,165559,2024-09-04T10:55:25.287,84,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.79,,1725440084039,true,428,2002,498,2043
18929485529,165559,2024-09-04T10:55:25.287,85,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.49,,1725440084039,true,356,2053,482,2064
18929485529,165559,2024-09-04T10:55:25.287,86,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,1.09,,1725440084039,true,931,2001,1001,2042
18929485529,165559,2024-09-04T10:55:25.287,87,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,0.69,,1725440084039,true,859,2052,985,2064
18929485529,165559,2024-09-04T10:55:25.287,88,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,UVP 8.99,,1725440085723,true,841,682,980,723
18929485529,165559,2024-09-04T10:55:25.287,89,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,5.85,,1725440085723,true,838,733,964,809
18929485529,165559,2024-09-04T10:55:25.287,90,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,UVP,,1725440085723,true,84,835,146,876
18929485529,165559,2024-09-04T10:55:25.287,91,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,CHANTRÉ Weinbrand,,1725440085723,true,84,887,445,935
18929485529,165559,2024-09-04T10:55:25.287,92,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,"je 0,7 I",,1725440085723,true,84,935,203,989
18929485529,165559,2024-09-04T10:55:25.287,93,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.79,,1725440085723,true,428,1533,498,1574
18929485529,165559,2024-09-04T10:55:25.287,94,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,2.49,,1725440085723,true,356,1584,482,1660
18929485529,165559,2024-09-04T10:55:25.287,95,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,10% gespart,,1725440085723,true,63,1686,255,1727
18929485529,165559,2024-09-04T10:55:25.287,96,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,PENNY Gouda-Scheiben,,1725440085723,true,63,1738,478,1786
18929485529,165559,2024-09-04T10:55:25.287,97,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 400 g,,1725440085723,true,63,1786,204,1840
18929485529,165559,2024-09-04T10:55:25.287,98,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,1.09,,1725440085723,true,931,1532,1001,1573
18929485529,165559,2024-09-04T10:55:25.287,99,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,11,android.widget.TextView,0.69,,1725440085723,true,859,1583,985,1659
18929485529,165559,2024-09-04T10:55:25.287,100,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,36% gespart,,1725440085723,true,567,1685,759,1726
18929485529,165559,2024-09-04T10:55:25.287,101,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,KINDER Überraschungs-Ei,,1725440085723,true,567,1737,1015,1785
18929485529,165559,2024-09-04T10:55:25.287,102,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,10,android.widget.TextView,je 20 g,,1725440085723,true,567,1785,686,1839
18929485529,165559,2024-09-04T10:55:25.287,103,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,Framstag,,1725440085723,true,263,276,462,333
18929485529,165559,2024-09-04T10:55:25.287,104,de,PENNY,de.penny.app,de.penny.app.main.view.MainActivity,,,9,android.widget.TextView,ab 06.09. bis 07.09.,,1725440085723,true,263,333,566,379
