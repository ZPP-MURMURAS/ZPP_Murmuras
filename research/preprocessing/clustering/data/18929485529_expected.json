{
    "coupons": [
        {
            "discount": "-33%",
            "new_price": "9.99",
            "old_price": "14.99",
            "product_name": "JOHNNIE WALKER Red Label Blended Scotch",
            "validity": "2024-09-07"
        },
        {
            "discount": "-24%",
            "new_price": "0.75",
            "old_price": "0.99",
            "product_name": "SAN MIGUEL Especial",
            "validity": "2024-09-07"
        },
        {
            "discount": "-7%",
            "new_price": "2.79",
            "old_price": "2.99",
            "product_name": "FELIX Knabber Mix 12 x 85 g",
            "validity": "2024-09-07"
        },
        {
            "discount": "-7%",
            "new_price": "2.79",
            "old_price": "3.89",
            "product_name": "FELIX Knabber Mix 200 g",
            "validity": "2024-09-07"
        },
        {
            "discount": "N/A",
            "new_price": "N/A",
            "old_price": "3.89",
            "product_name": "FELIX So gut wie es aussieht in Gelee",
            "validity": "2024-09-07"
        },
        {
            "discount": "-35%",
            "new_price": "5.85",
            "old_price": "8.99",
            "product_name": "CHANTRÉ Weinbrand",
            "validity": "2024-09-07"
        },
        {
            "discount": "-11%",
            "new_price": "2.49",
            "old_price": "2.79",
            "product_name": "PENNY Gouda-Scheiben",
            "validity": "2024-09-07"
        },
        {
            "discount": "-37%",
            "new_price": "0.69",
            "old_price": "1.09",
            "product_name": "KINDER Überraschungs-Ei",
            "validity": "2024-09-07"
        }
    ]
}